demo/attester/parsec-config.toml

```diff
@@ -21,7 +21,7 @@ auth_type = "UnixPeerCredentials"
 [[key_manager]]
 name = "sqlite-manager"
 manager_type = "SQLite"
-sqlite_db_path = "./kim-mappings/sqlite/sqlite-key-info-manager.sqlite3"
+sqlite_db_path = "/root/state/kim-mappings/sqlite/sqlite-key-info-manager.sqlite3"
```

Comment:
Yes, if you look in the `endorse.sh` script, before I start the `tpm_server` I move to this directory, so that's where the server creates its `NVChip` state as well!